make all failed due to none utils

$ make all                                         
PYTHONPATH=/home/abner/workspace/git/omnisearch/search pytest tests
================================================================================ test session starts ================================================================================
platform linux -- Python 3.6.9, pytest-5.3.5, py-1.8.1, pluggy-0.13.1
rootdir: /home/abner/workspace/git/omnisearch
collected 0 items / 1 error                                                                                                                                                         

====================================================================================== ERRORS =======================================================================================
_______________________________________________________________________ ERROR collecting tests/test_utils.py ________________________________________________________________________
ImportError while importing test module '/home/abner/workspace/git/omnisearch/tests/test_utils.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_utils.py:2: in <module>
    from common.utils import save_tmp_file
E   ModuleNotFoundError: No module named 'common.utils'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
================================================================================= 1 error in 0.04s ==================================================================================
Makefile:9: recipe for target 'test' failed
make: *** [test] Error 2

--- a/search/application/upload.py
+++ b/search/application/upload.py
@@ -7,7 +7,7 @@ from models.mapping import Mapping as DB
 from models.mapping import add_mapping_data
 from application.mapping import new_mapping_ins
 from common.config import MINIO_ADDR
-from common.util import save_tmp_file
+from common.utils import save_tmp_file
 
 
 def upload(name, **kwargs):

--- a/search/common/utils.py
+++ b/search/common/utils.py
@@ -21,7 +21,7 @@ def save_tmp_file(name, file_data=None, url=None):
                 extension = data_type.split(";")[0]
                 encode_method = data_type.split(";")[1]
                 if encode_method != "base64":
-                    raise DecodeError("Encode method not base64")
+                    raise DecodeError("Encode method not base64", Exception())
                 imgstring = img_data[1]
             else:
                 imgstring = img_data[0]

